lib/onc_certification_g10_test_kit/bulk_data_group_export_stu1.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'bulk_data_group_export'
+
+module ONCCertificationG10TestKit
+  class BulkDataGroupExportSTU1 < BulkDataGroupExport
+    title 'Group Compartment Export Tests STU1'
+    id :bulk_data_group_export_stu1
```

Comment:
This is changing the id of this group. I don't think it's worth making a new class just to change the title.